src/blocks/InfoCard.scss

```diff
@@ -46,3 +46,21 @@
     margin-bottom: var(--bloom-s4);
   }
 }
+
+.info-card__columns {
```

Comment:
Allows for the footer sections in the `AdditionalFees` component to be displayed as a grid instead of a vertical list